plugin.yaml

```diff
@@ -1,5 +1,5 @@
 name: "whatup"
-version: "0.2.1"
+version: "0.2.2"
```

Comment:
Can you remove this from the PR? The version is bumped when we are cutting a release. Since this is a feature addition it’d go into 0.3.0 anyways.

Reply:
done.